internal/usage/usage.go

```diff
@@ -196,6 +196,7 @@ const (
 	CASFilePath                     = "Path to a PEM file containing one or more CAs for database user authentication."
 	Verbose                         = "If true, returns all child jobs in the response."
 	ClusterID                       = "Unique identifier of the cluster."
+	RoleID                          = "Unique ID of the role to authorize."
```

Comment:
[q] I thought this command is for deauthorizing but see my other comment if we should instead not use a flag